storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java

```diff
@@ -471,11 +528,11 @@ public void exampleBlobURLParts() throws MalformedURLException, UnknownHostExcep
          Start with a URL that identifies a snapshot of a blob in a container and includes a Shared Access Signature
          (SAS).
          */
-        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?" +
-                "snapshot=2011-03-09T01:42:34.9360000Z" +
-                "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw" +
-                "&sip=168.1.5.60-168.1.5.70&spr=https,http&si=myIdentifier&ss=bf&srt=s" +
-                "&sig=92836758923659283652983562==");
+        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?"
+                + "snapshot=2011-03-09T01:42:34.9360000Z"
+                + "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw"
```

Comment:
Using StringBuilder would be near and efficient here

Reply:
I see your points, thanks for pointing this out. 
I prefer not to change right here for 2 reasons:
1. This is checkstyle change, so the improvement is kind of unrelated to the PR itself.
2. For StringBuilder, the Java JVM has the ability to optimize the code automatically, so it is not big concern of efficiency. Refer to this [one](https://stackoverflow.com/questions/4645020/when-to-use-stringbuilder-in-java).
My personal preference is not to introduce more structures to make code clean.